Src/EB/AMReX_EB2_3D_C.cpp

```diff
@@ -676,6 +690,11 @@ int build_faces (Box const& bx, Array4<EBCellFlag> const& cell,
 
             if (ncuts > 2) {
                 Gpu::Atomic::Add(dp,1);
+                if (plt_multiple_cuts){
+                    multicut_arr(i,j,k) = 10.0;
+                    // Not GPU friendly
+                    amrex::PrintToFile("loc_multicuts") << "fz: (x,y,z) = (" << problo[0]+(i)*dx[0] << ","<< problo[1]+(j)*dx[1] << "," << problo[2]+(k)*dx[2] << ")  \n";
```

Comment:
surround the PrintToFile bit with an `#ifndef AMREX_USE_GPU` so this can still compile on GPU.

Reply:
let me try to run things.